src/lib/validators.ts

```diff
@@ -81,11 +57,47 @@ export function isTshirtSize(classPart: string) {
 }
 
 export function isArbitraryShadow(classPart: string) {
-    const arbitraryValue = arbitraryValueRegex.exec(classPart)?.[1]
+    return getIsArbitraryValue(classPart, '', isValueShadow)
+}
+
+function getIsArbitraryValue(
+    classPart: string,
+    label: string,
+    testValue: (value: string) => boolean,
+) {
+    const result = arbitraryValueRegex.exec(classPart)
+
+    if (result) {
+        if (result[1]) {
+            return result[1] === label
+        }
 
-    if (arbitraryValue) {
-        return shadowRegex.test(arbitraryValue)
+        return testValue(result[2]!)
     }
 
     return false
 }
+
+function isValueLength(value: string) {
+    return lengthUnitRegex.test(value)
+}
+
+function isValueNever() {
+    return false
+}
+
+function isValueUrl(value: string) {
+    return value.startsWith('url(')
```

Comment:
This is missing `value.startsWith('url:')` which was previously present -- is that intentional ?

Reply:
Actually, I guess all the colons are dropped, due to the regex change -- ignore me!

Reply:
Yeah I check for the label in the second argument to `getIsArbitraryValue()`, the `isValueUrl` only needs to check the non-label part of the arbitrary value.